code-challenges.js

```diff
@@ -17,29 +17,91 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+// creat a function
+// two ranges to consider 0-212, 212-infanite
+// one point to consider 212
+// make a if else statment to log both ranges and set point
+// use ${} to actively log the input into the output with the statment
+
+const boilingPoint = (temp) => {
+  if (temp == 212) {
+    return `${temp} is at boiling point`
+  } else if (temp > 212) {
+    return `${temp} is above the boiling point`
+  } else if (temp < 212) {
+    return `${temp} is below the boiling point`
+  }
+}
+console.log(boilingPoint(42));
```

Comment:
The function looks great!  I would recommend running all of your variables through the function to make sure each condition is operating as expected.  To do this, you would need to make sure your variables are declared before the function and passed into your invocation as an arguement.